Queue/Queue.cpp

```diff
@@ -0,0 +1,86 @@
+#include"Queue.h"
+using namespace std;
+
+void CreateQueue(Queue& q, int size)
+{
+	q.data = new int[size];
+	q.capacity = size;
+	q.front = 0;
+	q.rear = -1;
+	q.count = 0;
+}
+
+
+int size(Queue& q)
+{
+	return q.count;
+}
+
+
+
+bool isFull(Queue& q)
+{
+	return (size(q) == q.capacity);
+}
+
+bool isEmpty(Queue& q)
+{
+	return (size(q) == 0);
+}
+
+int  EnQueue(Queue& q, int x)
+{
+	if (isFull(q))
+	{
+		cout << "Queue is Full";
+		return 0;
+	}
+	q.rear = (q.rear + 1) % q.capacity;
+	q.data[q.rear] = x;
+	q.count++;
+}
+
+
+int DeQueue(Queue& q)
+{
+	if (isEmpty(q))
+	{
+		cout << "Queue is Empty";
+		return 0;
```

Comment:
We should return -1 for errror, and 0 for success. 0 is a general success return code for function.

Reply:
I done